core/main.js

```diff
@@ -13,334 +13,9 @@
-  },
-  /**
-   * The main workspace most recently used.
-   * Set by Blockly.WorkspaceSvg.prototype.markFocused
-   * @name Blockly.mainWorkspace
-   * @type {Workspace}
-   * @suppress {checkTypes}
-   */
-  mainWorkspace: {
-    set: function (x) {
-      deprecation.warn(
-        'Blockly.mainWorkspace',
-        'version 9',
-        'version 10',
-        'Blockly.getMainWorkspace'
-      );
-      common.setMainWorkspace(x);
-    },
-    get: function () {
-      deprecation.warn(
-        'Blockly.mainWorkspace',
-        'version 9',
-        'version 10',
-        'Blockly.getMainWorkspace'
-      );
-      return common.getMainWorkspace();
-    },
-  },
-  /**
-   * Wrapper to window.prompt() that app developers may override to
-   * provide alternatives to the modal browser window. Built-in
-   * browser prompts are often used for better text input experience
-   * on mobile device. We strongly recommend testing mobile when
-   * overriding this.
-   * @name Blockly.prompt
-   * @type {!function(string, string, function()=)}
-   * @deprecated Use Blockly.dialog.prompt / .setPrompt() instead.
-   *     (December 2021)
-   * @suppress {checkTypes}
-   */
-  prompt: {
-    set: function (newPrompt) {
-      deprecation.warn(
-        'Blockly.prompt',
-        'version 9',
-        'version 10',
-        'Blockly.dialog.setPrompt'
-      );
-      dialog.setPrompt(newPrompt);
-    },
-    get: function () {
-      deprecation.warn(
-        'Blockly.prompt',
-        'version 9',
-        'version 10',
-        'Blockly.dialog.prompt'
-      );
-      return dialog.prompt;
-    },
-  },
-  /**
-   * Currently selected block.
-   * @name Blockly.selected
-   * @type {?ICopyable}
-   * @suppress {checkTypes}
-   */
-  selected: {
-    get: function () {
-      deprecation.warn(
-        'Blockly.selected',
-        'version 9',
-        'version 10',
-        'Blockly.getSelected'
-      );
-      return common.getSelected();
-    },
-    set: function (newSelection) {
-      deprecation.warn(
-        'Blockly.selected',
-        'version 9',
-        'version 10',
-        'Blockly.getSelected'
-      );
-      common.setSelected(newSelection);
-    },
-  },
-  /**
-   * The richness of block colours, regardless of the hue.
-   * Must be in the range of 0 (inclusive) to 1 (exclusive).
-   * @name Blockly.HSV_SATURATION
-   * @type {number}
-   * @suppress {checkTypes}
-   */
-  HSV_SATURATION: {
-    get: function () {
-      return colour.getHsvSaturation();
-    },
-    set: function (newValue) {
-      colour.setHsvSaturation(newValue);
-    },
-  },
-  /**
-   * The intensity of block colours, regardless of the hue.
-   * Must be in the range of 0 (inclusive) to 1 (exclusive).
-   * @name Blockly.HSV_VALUE
-   * @type {number}
-   * @suppress {checkTypes}
-   */
-  HSV_VALUE: {
-    get: function () {
-      return colour.getHsvValue();
-    },
-    set: function (newValue) {
-      colour.setHsvValue(newValue);
-    },
-  },
-});
```

Comment:
These were removed despite not being deprecated, and this has provoked [at least one question in the forum](https://groups.google.com/g/blockly/c/Zme8UGnh0Bg/m/9fxEJ9KEDAAJ).

@maribethb and @rachel-fenichel: I think we should restore these (with a `@deprecation` and `deprecation.warn()` and issue a patch release promptly, and re-remove them in a future version.